DOCKER.md

```
@@ -0,0 +1,147 @@
+## Monitoring
+
+### View Logs
+
+```bash
+docker-compose logs -f astra-guard
+docker-compose logs -f redis
+```
+
+### Access Dashboards
+
+- **Prometheus**: http://localhost:9090
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)